python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -231,3 +348,65 @@ def _get_function_call_from_chat_choice(self, choice: Union[Choice, ChunkChoice]
         if content.function_call is None:
             return None
         return FunctionCall(name=content.function_call.name, arguments=content.function_call.arguments)
+
+    def _build_streaming_message_with_tool_call(
+        self,
+        stream_chunks: Dict[int, List[OpenAIStreamingChatMessageContent]],
+        update_storage: Dict[str, Dict[int, Any]],
+    ) -> List[OpenAIStreamingChatMessageContent]:
+        """Build the streaming message with the tool call(s)."""
+        streaming_chat_message_contents = []
+        for index, _ in stream_chunks.items():
```

Comment:
Instead of `_` couldn't you store the value in `stream_chunk` and use it in line 361 instead of `stream_chunks[index]`?

Reply:
Thanks good suggestion. Tried to simplify the method a bit based on your feedback.